frontend/packages/dev-console/integration-tests/features/Helm/actions-on-HelmRelease.feature

```diff
@@ -0,0 +1,36 @@
+Feature: Helm Chart
+    User will be able to perform the actions on the helm releases
+
+Background:
+    Given user logged into the openshift application
+    And user is at developer perspecitve
+
+
+@regression, @smoke
+Scenario: Perform Upgrade action on Helm Release through Context Menu: HR-08-TC02
+    Given user is at the opened the context menu
```

Comment:
This scenarios works on topology page, So include "user is at topology page" in background section
and change the sentence something like "Given user opened the context menu"